.github/workflows/test-paradedb.yml

```diff
@@ -88,6 +89,8 @@ jobs:
       # In this environment, PARADEDB_TELEMETRY is set to false to avoid sending misleading telemetry data to PostHog.
       - name: Build the ParadeDB Docker Image via Docker Compose (dev only)
         if: steps.env.outputs.environment == 'dev'
+        with:
+          build-args: PG_VERSION_MAJOR=${{ matrix.pg_version }}
```

Comment:
You may need to add a new default to the `docker-compose` files too if we really need this. I would prefer if we didn't need a flag though and had a default value instead

Reply:
I'm also hesitating if we really need to pass a major version to the tests. Currently in dev branch, the tests will use the default version (which is now PG16). If we want to keep that, we may want to change the workflow title from PG17 back to PG16.

Reply:
Yeah. Let’s keep a default version and keep it on PG16